JitsiConference.js

```diff
@@ -2217,6 +2250,9 @@ JitsiConference.prototype._stopP2PSession
     // Stop P2P stats
     logger.info('Stopping remote stats with P2P connection');
     this.statistics.stopRemoteStats();
+    logger.info('Stopping CallStats for P2P connection');
```

Comment:
right above we have "with P2P connection" vs here "for P2P connection" let's pick one and be consistent. I'd say "for".